### PR TITLE
expanding acronyms in content

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/8ejaz38U/3422-add-definitions-for-acronyms-so-expander-can-be-removed

### Context

We're removing the acronym expander so we need to ensure that the content effectively explains any acronyms used.

### CODE REVIEW

**Comment by EllieNodder** [comment]
@ethax-ross not sure why this is failing - something to do with the welcome guide, I think? Could you take a look when you can thanks

File: app/views/content/train-to-be-a-teacher/who-do-you-want-to-teach.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ You could teach:
 - early years (children up to the age of 5)
 - further education (for example, teaching at further education colleges or sixth form colleges)
 
-If you want to [teach in primary or secondary schools](/train-to-be-a-teacher/if-you-have-a-degree) there are a number of ways to get [qualified teachers status](/what-is-qts) (QTS) which you will need to teach in the majority of schools in
+If you want to [teach in primary or secondary schools](/train-to-be-a-teacher/if-you-have-a-degree) there are a number of ways to get [qualified teacher status](/what-is-qts) (QTS) which you will need to teach in the majority of schools in
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/who-do-you-want-to-teach.md:18]:
```suggestion
If you want to [teach in primary or secondary schools](/train-to-be-a-teacher/if-you-have-a-degree) there are a number of ways to get [qualified teacher status (QTS)](/what-is-qts) which you will need to teach in the majority of schools in
```

Do we normally put QTS in the link text?

File: app/views/teaching_events/index.html.erb [hunk 2]
```diff
@@ -11,12 +11,12 @@
 
       <dl>
         <div>
-          <dt>DfE Train to Teach events</dt>
+          <dt>Department for Education (DfE) Train to Teach events</dt>
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/index.html.erb:14]:
```suggestion
          <dt>Train to Teach events</dt>
```

File: app/views/teaching_events/index.html.erb [hunk 3]
```diff
@@ -11,12 +11,12 @@
...
           <dd>Watch presentations, hear from teachers, speak to advisers and meet local training providers.</dd>
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/index.html.erb:15]:
```suggestion
          <dd>Watch presentations, hear from teachers, speak to advisers and meet local training providers. Run by the Department for Education (DfE).</dd>
```

Expanding the acronym in the title looks a bit clunky, especially on mobile, and distracts from the actual event type - I wonder whether this is a better alternative? Would also need applying to the Online Q&As as well if so.

**Review by gemmadallmandfe — COMMENTED** [review]
Looks great @EllieNodder - only a few minor comments but this should be good to go on Monday :-) 

There is only one page that I think you've missed - /train-to-be-a-teacher/initial-teacher-training has references to PGCE and PGDE that aren't defined!

File: app/views/content/steps-to-become-a-teacher/_step_4_find_the_right_training.html.erb [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <p>Explore your options, find out <a href="/train-to-be-a-teacher">how to train to
-  teach primary or secondary</a> (for example <a href="/what-is-a-pgce">a PGCE</a> or directly in a school).</p>
+  teach primary or secondary</a> (for example <a href="/what-is-a-pgce">a postgraduate certificate in education (PGCE)</a> or directly in a school).</p>
```
[the remark this answers is not in the document]
Comment (EllieNodder) [reply app/views/content/steps-to-become-a-teacher/_step_4_find_the_right_training.html.erb:2]: Will tackle the steps ticket next and make sure this reads a lot better!

File: app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md [hunk 5]
```diff
@@ -130,7 +130,7 @@ To get advice about training to teach, you can:
 
 #### Get help with international qualifications
 
-If your qualifications come from a non-UK institution, your teacher training provider may want to see a [statement of comparability](https://enic.org.uk/Qualifications/SOC/Default.aspx) from UK ENIC. A statement of comparability proves that your school and university qualifications are the same standard as UK GCSEs and a UK undergraduate degree.
+If your qualifications come from a non-UK institution, your teacher training provider may want to see a [statement of comparability](https://enic.org.uk/Qualifications/SOC/Default.aspx) from UK European Network of Information Centres (ENIC). A statement of comparability proves that your school and university qualifications are the same standard as UK GCSEs and a UK undergraduate degree.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md:133]: @EllieNodder Worth just flagging this one to Laura, explaining why we are doing this (but post-deploy)

File: app/views/content/train-to-be-a-teacher/initial-teacher-training.md [hunk 6]
```diff
@@ -66,7 +66,7 @@ Some courses begin with time spent in a university or school where you start you
 
 School placements vary in length and structure, but the minimum length is usually around 6 weeks. You should check with your training provider what their course and placement patterns are.
 
-Read about [Cindy’s first week as a trainee teacher doing a PGDE with QTS](/blog/my-first-week-as-a-trainee-teacher).
+Read about [what teacher training is like doing a QTS with PGCE at the University of Cumbria](/blog/what-to-expect-on-your-teacher-training).
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/initial-teacher-training.md:69]:
```suggestion
Read about [what teacher training is like doing a QTS with PGCE with one training provider](/blog/what-to-expect-on-your-teacher-training).
```

should we make this more generic to not promote Cumbria?

**Review by gemmadallmandfe — COMMENTED** [review]
One minor comment @EllieNodder - I'll get this merged once you've had a look

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**